Userland/Libraries/LibGfx/Gradients.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022, Daniel Oberhoff <daniel@danieloberhoff.de>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Vector.h>
+#include <LibGfx/Color.h>
+#include <LibGfx/Point.h>
+
+namespace Gfx {
+
+struct GradientStop {
+    Color color;
```

Comment:
Naming convention for data members,should,`m_` be used ?

Reply:
Not for structs, see https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#classes